src/main/java/seedu/duke/commands/AddCheatSheetCommand.java

```diff
@@ -34,7 +34,8 @@ public AddCheatSheetCommand(String nameOfFile) {
     @Override
     public void execute(UI ui) throws CommandException {
         Module module = ModuleList.getSelectedModule();
-        if (fileName.isEmpty() || sanitiseInput(fileName)) {
+        boolean isInvalidFileName = sanitiseInput(fileName);
+        if (fileName.isEmpty() || isInvalidFileName) {
```

Comment:
Maybe you can change the name of the method to isInvalidFileName instead as it is returning a boolean.

Reply:
Sure! As suggested in #218 .

Reply:
Sorry!  My bad!

Reply:
If possible I think the class name could be changed to a noun too e.g. FileNameVerifier? Maybe the class could be packaged somewhere else not under common since it is only used by AddCheatSheetCommand

Reply:
I mean this class can be used to sanitise input in the future. That's why it's in common. e.g. if we do not want to allow a user to include symbols in a task description, we can sanitise it first.

Reply:
Ok sure, then now I just change the class name to InputValidator? Or some other noun

Reply:
Ok! Thank u for the clarifications :)